app/services/plan_service/store/plan.rb

```diff
@@ -2,20 +2,26 @@ module PlanService::Store::Plan
 
   class TrialModel < ActiveRecord::Base
     self.table_name = :marketplace_trials
+    serialize :features, Array
```

Comment:
We may not need to store the `features` to the `marketplace_trials` table, because we know that these are trials and trials have certain set of features. To me this is a bit analogous to `plan_level`: We are not storing `plan_level` to `marketplace_trials` table, because we know that the value is always 0. Does that make sense?